src/CheckIfDead.php

```diff
@@ -219,30 +231,38 @@ protected function performFullRequest( $urls ) {
 	 * @return array Options for curl
 	 */
 	protected function getCurlOptions( $url, $full = false ) {
-		$header = [
-			// @codingStandardsIgnoreStart Line exceeds 100 characters
-			'Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5',
-			// @codingStandardsIgnoreEnd
-			'Cache-Control: max-age=0',
-			'Connection: keep-alive',
-			'Keep-Alive: 300',
-			'Accept-Charset: ISO-8859-1,utf-8;q=0.7,*;q=0.7',
-			'Accept-Language: en-us,en;q=0.5',
-			'Accept-Encoding: gzip,deflate',
-			'Pragma: '
-		];
+		$requestType = $this->getRequestType( $url );
+		if ( $requestType == "MMS" ) {
+			$url = str_ireplace( "mms://", "rtsp://", $url );
+		}
 		$options = [
 			CURLOPT_URL => $url,
 			CURLOPT_HEADER => 1,
 			CURLOPT_RETURNTRANSFER => true,
 			CURLOPT_AUTOREFERER => true,
 			CURLOPT_FOLLOWLOCATION => true,
 			CURLOPT_TIMEOUT => 30,
-			CURLOPT_USERAGENT => $this->userAgent,
 			CURLOPT_SSL_VERIFYPEER => false,
 			CURLOPT_COOKIEJAR => sys_get_temp_dir() . "checkifdead.cookies.dat"
 		];
-		$requestType = $this->getRequestType( $url );
+		if ( $requestType == "RTSP" || $requestType == "MMS" ) {
+			$header = [];
+			$options [CURLOPT_USERAGENT] = $this->mediaAgent;
+		} else {
+			$header = [
+				// @codingStandardsIgnoreStart Line exceeds 100 characters
+				'Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5',
+				// @codingStandardsIgnoreEnd
+				'Cache-Control: max-age=0',
+				'Connection: keep-alive',
+				'Keep-Alive: 300',
+				'Accept-Charset: ISO-8859-1,utf-8;q=0.7,*;q=0.7',
+				'Accept-Language: en-us,en;q=0.5',
```

Comment:
This seems a little dangerous to me. It's technically valid for a web server to return a 406 error code if it can't serve any of the languages listed in Accept-Language (although web servers almost never do this). It would be better to set it to 'Accept-Language: *', which means any language is accepted.